platform/appimage/koreader.appdata.xml

```diff
@@ -29,7 +30,7 @@
   <url type="homepage">https://koreader.rocks/</url>
   <url type="bugtracker">https://github.com/koreader/koreader/issues</url>
   <url type="faq">https://github.com/koreader/koreader/wiki</url>
-  <url type="translate">https://github.com/koreader/koreader#translation</url>
+  <url type="translate">https://hosted.weblate.org/engage/koreader/</url>
```

Comment:
You could also link to this one, but straight to Weblate should be fine.
https://github.com/koreader/koreader-translations